docs/tutorials/getting_started.rst

```diff
@@ -0,0 +1,290 @@
+===============
+Getting Started
+===============
+
+Installation
+============
+
+Qiskit Experiments is built on top of Qiskit, so we recommend that you first install
+Qiskit following its :external+qiskit:doc:`installation guide <getting_started>`. 
+Qiskit Experiments supports the same platforms and Python versions (currently 3.7+) as 
```

Comment:
Will we be aware about updating the Python version number here?

Reply:
Yes, PRs to update Python version in the future should update this string. It would be nice to pull it automatically from `setup.py` but that will be work for another PR.